packages/mui-joy/src/ChipDelete/ChipDelete.tsx

```diff
@@ -101,6 +104,27 @@ const ChipDelete = React.forwardRef(function ChipDelete(inProps, ref) {
 
   const classes = useUtilityClasses(ownerState);
 
+  const handleClickDelete = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    if (!disabled && onDelete) {
+      onDelete(event);
+    }
+    if (onClick) {
+      onClick(event);
+    }
+  };
+
+  const handleKeyDelete = (event: React.KeyboardEvent<HTMLButtonElement>) => {
+    event.preventDefault();
```

Comment:
What is the use case for `preventDefault()`?

Reply:
handleKeyDelete is firing twice for 1 keypress , e.preventDefault() is preventing that

Reply:
Then it should be specific for those keys. Currently, `TAB` is stuck on the chip delete.

https://user-images.githubusercontent.com/18292247/207773398-7adccae4-b9d9-4791-a7db-5c0111b86a3d.mov

Reply:
Got it so e.prevantDefault() should be inside if block, since you are committing to the pr will you make the change or should I.

Reply:
Got it, I can push the change.

Reply:
Sorry, I just pushed it. Hope it's fine

Reply:
working as expected now.
https://user-images.githubusercontent.com/60743144/207776339-9a1263da-add3-4298-b754-61eaed2237c1.mov